observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -153,41 +153,40 @@ include::snippets/technology-preview.adoc[]
-        tls: # <3>
+        endpoint: 0.0.0.0:8889 # <1>
+        tls: # <2>
           ca_file: ca.pem
           cert_file: cert.pem
           key_file: key.pem
-        namespace: prefix # <4>
-        const_labels: # <5>
+        namespace: prefix # <3>
+        const_labels: # <4>
           label1: value1
-        enable_open_metrics: true # <6>
-        resource_to_telemetry_conversion: # <7>
+        enable_open_metrics: true # <5>
+        resource_to_telemetry_conversion: # <6>
           enabled: true
-        metric_expiration: 180m # <8>
-        add_metric_suffixes: false # <9>
+        metric_expiration: 180m # <7>
+        add_metric_suffixes: false # <8>
     service:
       pipelines:
         metrics:
           exporters: [prometheus]
 # ...
 ----
-<1> Exposes the Prometheus port from the Collector pod and service. You can enable scraping of metrics by Prometheus by using the port name in `ServiceMonitor` or `PodMonitor` custom resource.
-<2> The network endpoint where the metrics are exposed.
-<3> The server-side TLS configuration. Defines paths to TLS certificates.
-<4> If set, exports metrics under the provided value. No default.
-<5> Key-value pair labels that are applied for every exported metric. No default.
-<6> If `true`, metrics are exported using the OpenMetrics format. Exemplars are only exported in the OpenMetrics format and only for histogram and monotonic sum metrics such as `counter`. Disabled by default.
-<7> If `enabled` is `true`, all the resource attributes are converted to metric labels by default. Disabled by default.
-<8> Defines how long metrics are exposed without updates. The default is `5m`.
-<9> Adds the metrics types and units suffixes. Must be disabled if the monitor tab in Jaeger console is enabled. The default is `true`.
+<1> The network endpoint where the metrics are exposed (required).
+<2> The server-side TLS configuration. Defines paths to TLS certificates.
+<3> If set, exports metrics under the provided value. No default.
+<4> Key-value pair labels that are applied for every exported metric. No default.
+<5> If `true`, metrics are exported using the OpenMetrics format. Exemplars are only exported in the OpenMetrics format and only for histogram and monotonic sum metrics such as `counter`. Disabled by default.
+<6> If `enabled` is `true`, all the resource attributes are converted to metric labels by default. Disabled by default.
+<7> Defines how long metrics are exposed without updates. The default is `5m`.
+<8> Adds the metrics types and units suffixes. Must be disabled if the monitor tab in Jaeger console is enabled. The default is `true`.
+
+Note the port specified in the `endpoint` field is exposed automatically by the operator in the `<instance name>-collector` service.
+
+Also, you can create automatically the *ServiceMonitor* or *PodMonitor* to help Prometheus to scrape your metrics by setting the `spec.observability.metrics.enableMetrics` field to `true`.
```

Comment:
This paragraph doesn't seem to be part of this topic, so I suggest turning it into a note admonition:
```suggestion
[NOTE]
====
When the `spec.observability.metrics.enableMetrics` field in the `OpenTelemetryCollector` custom resource (CR) is set to `true`, the `OpenTelemetryCollector` CR automatically creates a Prometheus `ServiceMonitor` or `PodMonitor` CR to enable Prometheus to scrape your metrics.
====
```